tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -464,12 +464,24 @@ Duration <DURATION>"
 
 exports[`bring_invalid_dir.w 1`] = `
 "error: Cannot explicitly bring directory with no Wing files "no_wing_files"
+  --> ../../../examples/tests/invalid/bring_invalid_dir.w:4:7
+  |
+4 | bring "./subdir3/no_wing_files" as whatchaGonnaBring;
+  |       ^^^^^^^^^^^^^^^^^^^^^^^^^
 
 
-error: Cannot bring Wing directories that contain invalid characters: "in.valid"
+error: Cannot bring Wing directories that contain sub-directories with invalid characters: "in.valid"
+  --> ../../../examples/tests/invalid/bring_invalid_dir.w:1:7
+  |
+1 | bring "./subdir3" as wontWork;
+  |       ^^^^^^^^^^^
```

Comment:
Can we add a "hint" annotation that says what characters are allowed?  Or maybe replace "invalid" with "nonalphanumeric" - either can work